README.md

```diff
@@ -3,18 +3,18 @@
 [![GitHub tag (latest SemVer incl. pre-release)](https://img.shields.io/github/v/tag/edaa-org/pySystemVerilogModel?logo=GitHub&include_prereleases)](https://github.com/edaa-org/pySystemVerilogModel/tags)
 [![GitHub release (latest SemVer incl. including pre-releases)](https://img.shields.io/github/v/release/edaa-org/pySystemVerilogModel?logo=GitHub&include_prereleases)](https://github.com/edaa-org/pySystemVerilogModel/releases/latest)
 [![GitHub release date](https://img.shields.io/github/release-date/edaa-org/pySystemVerilogModel?logo=GitHub&)](https://github.com/edaa-org/pySystemVerilogModel/releases)
-[![Dependent repos (via libraries.io)](https://img.shields.io/librariesio/dependent-repos/pypi/pySystemVerilogModel?logo=GitHub)](https://github.com/edaa-org/pySystemVerilogModel/network/dependents)  
+[![Dependent repos (via libraries.io)](https://img.shields.io/librariesio/dependent-repos/pypi/pySystemVerilogModel?logo=GitHub)](https://github.com/edaa-org/pySystemVerilogModel/network/dependents)
```

Comment:
Your editor killed Markdown linebreaks. (3x in total)

Isn't is using the settings from `.editorconfig`?